### ISSUE TITLE
Parcel not found

### ISSUE THREAD

**Issue by a1300** [vlucas/vlid#11]
Reproduction

```bash
# clone
git clone https://github.com/vlucas/vlid

# cd
cd vlid

# install dependencies
npm install

# build
npm run build
# -> sh: 1: parcel: not found

```

**Comment by vlucas** [comment]
Parcel is the bundler, which in this case was installed globally. You should not need it unless you are building the package for release.

You can fix this with `npm i -g parcel-bundler`.

You do perhaps have a point that it should be included in the `devDependencies` list though.

**Comment by a1300** [comment]
Yes, I would add it to the `devDependencies` also to fixate the `parcel` version
